### ISSUE TITLE
Can't sign in using email address under Firefox on Android

### ISSUE THREAD

**Issue by ndarilek** [sandstorm-io/sandstorm#2017]
If I'm using Firefox on Android, tapping the email field to sign in doesn't make the keyboard appear. The only way I've been able to sign in is if I use another computer, enter my email address, then click the link I'm sent on the Android device.

Initially I thought this was due to incorrect ARIA markup that used to exist on these fields, but that doesn't appear to be the case even with the markup gone.

It's also possible that this is due to an issue fixed in Firefox. Unfortunately I'm stuck on Firefox 42 on Android due to a critical a11y regression. I don't have any specific idea what this issue might be, but I'd be remiss if I didn't admit to using an old Firefox.

I also thought it might be due to having TalkBack enabled, but when I disabled it and tried to have my girlfriend tap the field, the keyboard didn't appear for her either. It's possible that Firefox changes how it interacts when TalkBack is enabled, but doesn't deactivate that change when TalkBack is temporarily disabled. I doubt that but am just spitballing possibilities.

Also, I don't have this issue on any other fields. It's just the email address field when attempting to sign in.

Thanks.

**Comment by paulproteus** [comment]
Hi Nolan,

That's pretty weird and sad. Are you still experiencing this? If so, I can
see if I can reproduce it.

**Comment by ndarilek** [comment]
Just tried now and it still doesn't work for me.

**Comment by paulproteus** [comment]
Good to know. I'll queue a task to look into it.

**Comment by ndarilek** [comment]
Tried this recently and it seems to have gone away. Probably should have mentioned that I was stuck on Firefox 42 because of accessibility issues. When I tried this again under 48 or whatever is current, it seemed to work. Firefox went through a brief period where it interacted poorly with Android accessibility, and I of course couldn't disable said accessibility to check if that was the issue here. :)
